src/data/analytics.js

```diff
@@ -47,7 +47,6 @@ export const createRequestByFab    = () => trackMenuEvent('create request', 'fab
 export const createRequestByButton = () => trackMenuEvent('create request', 'button')
 export const createRequestByTile   = () => trackMenuEvent('create request', 'tile')
 export const clickedLogo           = () => trackMenuEvent('logo clicked')
-export const clickedHelp           = () => trackMenuEvent('help')
```

Comment:
Do we have a way to still collect this metric with the new help widget?

Reply:
I looked for a way, and the only answer I could come up with would be to create our own fab which opens the widget. I wasn't prepared to put that much work into it right now.

Reply:
`<Help on:mouseup={clickedHelp} />` wont work then?

Reply:
I see what you are saying now. You would need an element to fire the event.